### ISSUE TITLE
Compilation error in PacketDrillApp

### ISSUE THREAD

**Issue by ApolloLV** [inet-framework/inet#251]
I tried compiling inet on my machine, but I got some compilation errors and the build was unable to finish.
Changing the lines 939 and following in inet/applications/packetdrill/PacketDrillApp.cc to 

`if (expr_params->spp_hbinterval->getNum() > 0)
                sctpSocket.setHbInterval(expr_params->spp_hbinterval->getNum());
            if (expr_params->spp_pathmaxrxt->getNum() > 0)
                sctpSocket.setPathMaxRetrans(expr_params->spp_pathmaxrxt->getNum());`
got rid of the compilation error, however I am not able to confirm whether or not PacketDrillApp will execute as intended using those changes.
Did anybody else encounter this error?
Is my temporary fix correct?

Full compiler output below
-----
`[user@machine src]$ make
*** COMPILING with:
clang++ -c -std=c++11 -g -Wall -MMD -MP -MF .d  -fPIC  -Wno-deprecated-register -Wno-unused-function -fno-stack-protector -DHAVE_SWAPCONTEXT -DWITH_MPI -DXMLPARSER=expat -DPREFER_QTENV -DWITH_QTENV -DWITH_TKENV -DWITH_PARSIM -DWITH_NETBUILDER -DWITH_OSG -DWITH_OSGEARTH -Wno-overloaded-virtual -include inet/common/precompiled.h   -I. -I/opt/omnetpp/include
*** LINKING with:
clang++ -shared -fPIC -o ../out/clang-debug/src/libINET.so -Wl,--no-as-needed -Wl,--whole-archive  -Wl,--no-whole-archive -loppenvird -loppsimd -ldl -lstdc++  -lOpenThreads -losg -losgText -losgDB -losgEarth -losgEarthUtil -Wl,-rpath,/opt/omnetpp/lib -Wl,-rpath,/lib -Wl,-rpath,.  -L/opt/omnetpp/lib
Building...
inet/applications/packetdrill/PacketDrillApp.cc
inet/applications/packetdrill/PacketDrillApp.cc:939:45: error: ordered comparison between pointer and zero
      ('PacketDrillExpression *' and 'int')
            if (expr_params->spp_hbinterval > 0)
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~
inet/applications/packetdrill/PacketDrillApp.cc:941:45: error: ordered comparison between pointer and zero
      ('PacketDrillExpression *' and 'int')
            if (expr_params->spp_pathmaxrxt > 0)
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~
2 errors generated.
make: *** [Makefile:1162: ../out/clang-debug/src/inet/applications/packetdrill/PacketDrillApp.o] Error 1
`

**Comment by LongHairedHacker** [comment]
I have encountered the same problem and use the same fix.
At least it compiles for now.
Judging from the surrounding code it should be the correct fix,
but I've never worked with that part of INET so I can't be sure.

**Comment by rhornig** [comment]
Correct. This is already fixed on master in 9df2c1ae82fa4b2fa21651a674a19d25ec47a47b
